src/main/java/core/basesyntax/dao/FruitsDao.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.FruitRecord;
+import java.util.List;
+
+public interface FruitsDao {
+    boolean addRecord(FruitRecord fruitRecord);
+
+    List<FruitRecord> getRecords();
+
+    FruitRecord getRecord(FruitRecord newFruitRecord);
+
+    boolean modifyRecord(FruitRecord fruitRecord);
+
+    boolean isRecordExists(FruitRecord fruitRecord);
```

Comment:
could make this method private and delete it from interface

Reply:
Fixed.